corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -7,6 +7,7 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
     var FormplayerFrontend = hqImport("cloudcare/js/formplayer/app");
     var separator = " to ";
         dateFormat = "YYYY-MM-DD";
+    var selectDelimiter = "#,#"; // Formplayer also uses this
```

Comment:
I couldn't find this in formplayer or core code, where is it referenced?

Reply:
Found in new code here: https://github.com/dimagi/commcare-core/pull/998/files#diff-8e9d0b0d9bfa096b6eaa56611b7bb11f5b0afc3050469fb5ed54bfcd61ef1103R38

Reply:
Here https://github.com/dimagi/commcare-core/pull/998/files#diff-8e9d0b0d9bfa096b6eaa56611b7bb11f5b0afc3050469fb5ed54bfcd61ef1103R38

And here is a test https://github.com/dimagi/formplayer/pull/973/files#diff-fe74f2d74e4a73bed2a3e0fe04087d7d48aeb94da5e6d321471d8b47396d0fa0R134